TypeCobol.Analysis.Test/ViolationTest.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System.Runtime.CompilerServices;
+using System.Text;
+using TypeCobol.Test;
+using static TypeCobol.Analysis.Test.CfgTestUtils;
+using static TypeCobol.Compiler.Symbols.Symbol;
```

Comment:
Why the 2 `using static` ? Please remove them.

Reply:
Removed